src/javascripts/Factories/contextMenuFactory.js

```diff
@@ -130,6 +130,9 @@ ngapp.service('contextMenuFactory', function() {
                     label: 'Change Description',
                     hotkey: 'Alt+Shift+D',
                     callback: () => scope.changeFileDescription(node)
+                }, {
+                    label: 'Add Masters',
```

Comment:
I don't think this should be in the refactor submenu.  Perhaps we should make a new masters submenu where we will have the options:

- Add Masters
- Sort Masters
- Clean Masters

?

(the other two can be disabled for the time being)